Makefile.am

```diff
@@ -28,4 +28,4 @@ libtool: $(LIBTOOL_DEPS)
 # Needed by autoconf because we use README.md instead of README.
 # See http://stackoverflow.com/q/15013672/
 README: README.md
```

Comment:
My bad. autogen.sh would also need to be updated to pass `--foreign` to automake to stop it from complaining about the names.
These are very silly warnings anyway.